add-ons/bp-search-block/bp-search-block.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * A BuddyPress Block to search for activities, members, groups, sites, posts from any post, page or widget of your community site!
```

Comment:
> posts from any post?

Reply:
Well yes 😄 , it's actually what can happen if the block is inserted into a post. I'll replace by "content"